### ISSUE TITLE
Modal: Add hidden class

### ISSUE THREAD

**Issue by matthieu-castier** [adeo/mozaic-design-system#863]
# 🚀 Styleguide feature request
Add a hidden class to prevent a bug when several modals are present.

### Description
When there are several modals in a page, they overlap in the code and prevent from clicking on the close button. 

### Describe the solution you'd like

Add : 
```
.mc-modal {
    &.is-hidden {
        visibility: hidden;
    }
}
```

### Describe alternatives you've considered
 Have you considered any alternative solutions or workarounds?

**Comment by tiloyi** [comment]
Hello @matthieu-castier ,
A behavior similar to what you propose has already been implemented in the last Mozaic release (version 1.12.0) via the following PR => https://github.com/adeo/mozaic-design-system/pull/855

Does this solve your problem? Or still not?

**Comment by matthieu-castier** [comment]
Hello @tiloyi, no it doesn't fix the modal overlay problem.

**Comment by tiloyi** [comment]
Oh okay, I got it.
I'll do the fix now. It will be in tomorrow's release.

**Comment by tiloyi** [comment]
Hello @matthieu-castier ,
As seen together, after testing, the problem is fixed via the PR #855.

I close this ticket.
Don't hesitate to come back to us if you encounter other problems.
